go/mysql/bufio_pool_test.go

```diff
@@ -0,0 +1,46 @@
+package mysql
+
+import (
+	"bufio"
+	"bytes"
+	"testing"
+)
+
+func TestWriter(t *testing.T) {
```

Comment:
can you add a test where you have multiple writers. give them each an identifying string to write, and make sure that we don't have corrupt or unclean pooling behavior.